cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,42 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until the database is available."""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
```

Comment:
It would be more Pythonic to use `None` to represent a failed connection attempt rather than an unassigned variable. Consider initializing `db_conn` to `None`.